app/components/page_heading/view.rb

```diff
@@ -0,0 +1,29 @@
+class PageHeading::View < GovukComponent::Base
+  attr_accessor :title, :errors
+  include ActiveModel
+  def initialize(title: "", errors: false)
+    @title = title
+    @errors = errors.present?
+  end
+
+  def build_page_title
+    [build_error + build_title, build_service_name, "GOV.UK"].join(" - ")   
+  end
+
+  def build_page_heading
+    build_title
+  end
```

Comment:
I find this hard to follow - it's a bit circular. 

`build_page_heading` calls `build_title` that then calls `page_headings` 

I suspect page heading should just use whatever page heading is defined int the translation file - there's nothing to build.

Page titles should then either use that, or define a new one.

Reply:
I've changed it so that the page title component now draws from its own section of the locales now.